spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -64,16 +67,18 @@ def get_parser():
         '-i',
         metavar=Metavar.file,
         required=True,
-        help='Spinal cord segmentation mask to compute morphometrics. Example: sub-001_T2w_seg.nii.gz'
-             '\nNote: If no normalization is wanted (i.e., if the "-normalize" flag is not specified),'
+        help='Spinal cord or spinal canal segmentation mask to compute morphometrics from. If spinal cord segmentation '
+             'is provided, MSCC is computed. If spinal canal segmentation is provided, MCC is computed. '
```

Comment:
I would maybe tweak the wording here to mention the CSF in addition to the spinal canal:

```suggestion
        help='Spinal cord or spinal canal segmentation mask to compute morphometrics from. If spinal cord segmentation '
             'is provided, MSCC is computed. If a spinal canal (i.e. CSF) mask is provided, MCC is computed instead.'
```

Reply:
see https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4331/files#r1447820600